cmake/developer_package/api_validator/api_validator.cmake

```diff
@@ -111,7 +111,7 @@ function(_ov_add_api_validator_post_build_step)
     else()
         message(FATAL_ERROR "Unknown configuration: ${CMAKE_HOST_SYSTEM_PROCESSOR}")
     endif()
-
+    message("find path: ${PROGRAMFILES}/Windows Kits/10/build/${CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION}/universalDDIs/${wdk_platform}")
```

Comment:
what issue you are trying to solve?

FYI https://github.com/openvinotoolkit/openvino/pull/25252

Reply:
I'm trying to figure out why I can't find the UniversalDDIs.xml file during compilation with OV commit after 169eeec48f57ddf14ed7226c564f10d3fca005a8, and I'll fix this by patching, won't merge